GameBoard/Gomoku/GomokuBoard.cs

```diff
@@ -0,0 +1,113 @@
+using System.Data;
+using System.Security.Cryptography;
+using System.Text.Json;
+using GameBoard;
+
+namespace Gomoku;
+
+public class GomokuBoard : GameBoard.GameBoard
+{
+    protected override int PlayerCount => 2;
+    protected override string GameRecordFileName => "gomoku-record.json";
+    public override string GameName => "Gomoku";
+    private const char FirstPlayerSymbol = 'o';
+    private const char SecondPlayerSymbol = 'x';
+
+    static GomokuBoard()
+    {
+        GameBoardFactory.RegisterGame(() => new GomokuBoard());
+    }
+    
+    protected override void SetupGameBoard()
+    {
+        Size = 15;
+
+        base.SetupGameBoard();
+    }
+
+    public override bool CheckWin(int row, int col, object value = null)
+    {
+        const int minEdge = 0;
+        const int winningCount = 5;
+        if (value == null)
+        {
+            value = Board[row, col];
+            if (value == null) { return false; }
```

Comment:
How about using `NotPlacedFlag` rather than `null`? If we use the defined const value, we don't have to modify the value if we decide to change the default value of each cell in the future.
```suggestion
            if (value == null) { return false; }
            if (value == NotPlacedFlag) { return false; }
```